src/styles/general.scss

```diff
@@ -0,0 +1,14 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
it's only recommended to use `box-sizing` at `*` selector. Move other rules to `html`/`body` tag selectors

Reply:
If I set margin: 0 on the body element, it won't affect the margins of its child elements